eng/test.yaml

```diff
@@ -11,13 +11,13 @@ steps:
   inputs:
     command: custom
     projects: |
-      src/**/*.Tests.csproj
+      $(Build.SourcesDirectory)\arcade-services.sln
     custom: test
     arguments: > 
       --configuration $(_BuildConfig)
       --collect:"Code Coverage"
       --settings:CodeCoverage.runsettings
-      --filter "TestCategory!=PostDeployment&TestCategory!=Nightly&TestCategory!=PreDeployment"
```

Comment:
Only comment here is to make sure we update the Validation documentation with this new test category and how it should be used. 

https://github.com/dotnet/core-eng/tree/master/Documentation/Validation#test-categorization

Reply:
Those other categories don't appear to exist anywhere in arcade-services.  I could re-target all those tests to be "PostDeployment" I suppose. :-) So make them line up with the documentation.

Reply:
Up to you :)  It's a living thing, so we shouldn't feel confined to only these, as these made sense at the time I documented them. (Was to help document when and where the tests would run, rather than what they were)

Reply:
Retargeted!